File: C++11/백준/Bronze/4635. Speed Limit/Speed Limit.cc
#include <iostream>
using namespace std;

int main(){
    int a,b,c;
    while(true){
        cin >> a;
        if (a==-1)break;
        int s = 0;
        int ans = 0;
        for (int i=0;i<a;i++){
            cin >> b >> c;
            ans += b*(c-s);
            s+=c-s;
        }
        cout << ans <<" miles" << endl;
    }
}

File: Java/백준/Bronze/4635. Speed Limit/Speed Limit.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        while(true){
            int a = in.nextInt();
            if (a==-1){break;}
            int s = 0;
            int ans = 0;
            for (int i=0;i<a;i++){
                int b = in.nextInt();
                int c = in.nextInt();
                ans += b*(c-s);
                s += c-s;
            }
            System.out.printf("%d miles\n",ans);
        }
    }
}